src/alto-ui/Tag/Tag.scss

```diff
@@ -0,0 +1,48 @@
+@import '../scss/inc';
+
+.Tag {
+  background-color: $blue-10;
+  border: 1px solid $blue-20;
+  color: $blue;
+  border-radius: $border-radius-default;
+  padding: $spacing-x-small $spacing-small;
+  display: flex;
+  align-items: center;
+  line-height: 1;
+
+  &--button {
+    @extend %reset-button;
+    transition: all $transition;
```

Comment:
`transition all` should be avoided because actually you animate only `border-color` use transition property or write this specific property, it's not necessary to animate all.

Reply:
it's kind of a default behavior

Reply:
yep, but we can narrow that